rust-connectors/sources/http/tests/get-test-json.bats

```diff
@@ -30,6 +30,6 @@ teardown() {
     sleep 10
 
     run fluvio consume -o 1 --end-offset 1 -d $TOPIC
-    assert_output --partial '{"body": {"Hello, Fluvio! - ' 
+    assert_output --partial '{"body":"Hello, Fluvio! - ' 
```

Comment:
I think that we should remove the initial `{`

Reply:
it found an actual bug. the status should not be there with output_parts body

Reply:
yeah Mondays... :innocent: